frontend/apps/picasso/pages/transfers/index.tsx

```diff
@@ -146,6 +164,11 @@ const Transfers: NextPage = () => {
         <Grid item {...gridItemStyle("1.5rem")}>
           <TransferRecipientDropdown />
         </Grid>
+        {isKSMAlertVisible ? (
+          <Grid item {...gridItemStyle("1.5rem")}>
+            <TransferKSMAlert />
+          </Grid>
+        ) : null}
```

Comment:
```suggestion
        {isKSMAlertVisible && (
          <Grid item {...gridItemStyle("1.5rem")}>
            <TransferKSMAlert />
          </Grid> 
        )}
```
unecessary ternary

Reply:
Conditional rendering with logical operators has a tendency to create issues later on if we don't be careful with it to make sure it is always boolean. There is always the danger of short-circuiting when the parent condition changes. 

That's why I have chosen the safest, but, more verbose route.